core/src/main/java/com/microsoft/applicationinsights/internal/config/TelemetryProcessorCreator.java

```diff
@@ -46,7 +48,48 @@ public TelemetryProcessor Create(TelemetryProcessorXmlElement confClass) {
             return null;
         }
 
+
+        // If the <ExcludedTypes> tag is not empty
```

Comment:
See how it's done in .NET. Configuration reader can detect that the property you are about to initialize is a list and just adds values to it

Reply:
No hardcoded names